docs/changelog.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../CHANGELOG.rst
```

Comment:
Another option here is to move all of the documentation into the `docs`. I believe GitHub knows to look for such a folder when rendering the readme and licensing info

Reply:
that's good to know, thanks! I'm inclined to keep this structure (top-level) for now, so that the typical items are relatively more visible/discoverable